cuda_bindings/tests/test_nvjitlink.py

```diff
@@ -53,71 +76,82 @@ def test_invalid_arch_error():
 
 
 def test_create_and_destroy():
-    handle = nvjitlink.create(1, ["-arch=sm_53"])
-    assert handle != 0
-    nvjitlink.destroy(handle)
+    for option in ARCHITECTURES:

```

Comment:
Let's use `@pytest.parametrize` for this. One less indentation is good (so is less diff to review)